.github/workflows/build-tarball.yml

```diff
@@ -25,13 +25,12 @@ on:
       - '!.github/workflows/build-tarball.yml'
 
 env:
+  PYTHON_VERSION: '3.10'
```

Comment:
Do we need the quotes here?

Reply:
No, but I leave them for the other PR #41756.

Reply:
I think we do. Otherwise it's parsed as the number 3.1

Reply:
Maybe we should require quotes everywhere after all, YAML is not parsable by humans 😬